src/ResourceManagement/Scheduler/Microsoft.Azure.Management.Scheduler/Generated/Models/BasicAuthentication.cs

```diff
@@ -33,7 +33,7 @@ public BasicAuthentication(HttpAuthenticationType? type = default(HttpAuthentica
         public string Username { get; set; }
```

Comment:
Not sure why we need to change generate.cmd.  This isn't a breaking change.  This is a bug fix, we had it wrong in the previous swagger/SDK and we are fixing it righ tnow!